geostore/tiles/helpers.py

```diff
@@ -108,7 +108,7 @@ def pixel_widths(self, xmin, ymin, xmax, ymax):
         )
 
     @cached_tile
-    def get_tile(self, x, y, z):
+    def get_tile(self, x, y, z, name=None, features_pks=[]):
```

Comment:
```suggestion
    def get_tile(self, x, y, z, name=None, features_pks=None):
        if not features_pks:
            features_pks = []
```

BEst practice to take care about mutation in default values (list and dict)